eng/pipelines/templates/stages/archetype-sdk-client.yml

```diff
@@ -53,6 +53,9 @@ parameters:
 - name: MatrixReplace
   type: object
   default: []
+- name: SpellCheckPublicApiSurface
+  type: boolean
+  default: false
```

Comment:
Why not default to true?

Reply:
Per the corresponding email, we need to get clean first.